package.json

```diff
@@ -105,19 +98,19 @@
         "karma-jasmine": "1.1.1",
         "karma-jasmine-html-reporter": "^0.2.2",
         "lint-staged": "^7.1.2",
-        "ng-packagr": "~2.4.1",
+        "ng-packagr": "~4.1.1",
         "node-sass": "4.8.3",
         "path": "^0.12.7",
         "prettier": "^1.12.1",
         "protractor": "~5.3.1",
         "rxjs-tslint": "^0.1.4",
         "semantic-release": "^15.9.5",
         "ts-node": "^5.0.1",
-        "tsickle": ">=0.25.5",
-        "tslib": "^1.7.1",
+        "tsickle": ">=0.32.0",
+        "tslib": "^1.9.3",
         "tslint": "^5.8.0",
         "tslint-config-prettier": "^1.13.0",
-        "typescript": "2.7.2",
+        "typescript": "2.9.2",
```

Comment:
Why use latest type script when angular explicitly states: `The Angular Compiler requires TypeScript >=2.7.2 and <2.10.0 but 3.0.3 was found instead.`